templates/documentation.html

```diff
@@ -69,16 +69,6 @@ <h3 class="text-center">
                   </a>
                 </div>
               </div>
-              <div class="list-group">
-                <a class="list-group-item" href="https://inveniordm.docs.cern.ch/deployment/">
-                  <i class="fa fa-fw fa-rocket"></i>
-                  Deploy
-                </a>
-                <a class="list-group-item" href="https://inveniordm.docs.cern.ch/extensions/">
-                  <i class="fa fa-fw fa-cubes"></i>
-                  Extend
```

Comment:
The `Extensions` section is here https://inveniordm.docs.cern.ch/develop/topics/extensions/